sdcm/sct_config.py

```diff
@@ -713,6 +713,15 @@ class SCTConfiguration(dict):
         dict(name="azure_image_username", env="SCT_AZURE_IMAGE_USERNAME", type=str,
              help=""),
 
+        dict(name="azure_root_disk_size_monitor", env="SCT_AZURE_ROOT_DISK_SIZE_MONITOR", type=int,
+             help=""),
+
+        dict(name="azure_root_disk_size_db", env="SCT_AZURE_ROOT_DISK_SIZE_DB", type=int,
+             help=""),
+
+        dict(name="azure_root_disk_size_loader", env="SCT_AZURE_ROOT_DISK_SIZE_LOADER", type=int,
+             help=""),
```

Comment:
as in my previous comment, i think it will make sense to use same parameter as used for other backends, as AFAICT we won't use mixed backends, and the name is self explanatory (and holds only the size number), making a new parameter to be unnecessary..

Reply:
I agree, separate trello task created as in the commend above.